Dockerfile

```diff
@@ -4,17 +4,22 @@ ENV TZ=America/Chicago
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 
 RUN apt upgrade -y
-RUN apt update && apt install -y sudo wget gnupg2 git gcc gfortran libboost-dev bzip2 openmpi-bin flex build-essential bison libboost-all-dev vim libsqlite3-dev numactl sqlite3 gdb
-RUN wget -q -O - https://repo.radeon.com/rocm/rocm.gpg.key | sudo apt-key add -
-RUN echo 'deb [arch=amd64] https://repo.radeon.com/rocm/apt/debian/ ubuntu main' | sudo tee /etc/apt/sources.list.d/rocm.list
-RUN apt update
-RUN apt install -y rocm-dev6.0.0 rocm-libs6.0.0 build-essential libssl-dev swig numpy-stl
+RUN apt update && apt install -y sudo wget gnupg2 git gcc gfortran libboost-dev bzip2 openmpi-bin flex build-essential bison libboost-all-dev vim libsqlite3-dev numactl sqlite3 gdb libgtest-dev
+
+#Rocm 6.1.1
```

Comment:
ROCm, not Rocm :-)

Reply:
Will fix

Reply:
Fixed: e18bd4c176800c0ce3c246d380bdb788e2fb94a0